src/main/resources/com/google/api/codegen/ruby/method_sample.snip

```diff
@@ -0,0 +1,134 @@
+@extends "ruby/common.snip"
+
+@snippet sampleCode(apiMethod)
+    @@example
+    @let lines = util.getDocLines(sampleCode_(apiMethod))
+        @join line : lines
+            {@""}  {@line}
+        @end
+    @end
+@end
+
+@snippet sampleCode_(apiMethod)
+    require "{@apiMethod.initCode.apiFileName}"
+
+    {@apiMethod.apiClassName} = {@apiMethod.initCode.packageName}\
+        ::{@apiMethod.apiClassName}
+    @join aliasDef : context.entrySet(apiMethod.initCode.typeTable.getImportsMap)
```

Comment:
Don't use 'context' in snippets implemented with MVVM, it will then require extra work later when completely converting the rest of Ruby snippets to MVVM. In this particular case, when you convert to using List<ImportTypeView>, you won't need the entrySet() call any more.